backend/routes.js

```diff
@@ -168,4 +171,176 @@ module.exports = function routes(app, logger) {
       }
     });
   });
+
+  // POST /fixed (create a fixed auction)
+  app.post('/fixed', (req, res) => {
+      pool.getConnection(function (err, connection) {
+        if(err) {
+          // if there is an issue obtaining a connection, release the connection instance and log the error
+          logger.error("Problem obtaining MySQL connection", err);
+          res.status(400).send("Problem obtaining MySQL connection");
+        } else {
+          // if there is no issue obtaining a connection, execute query and release connection
+          jwt.verifyToken(req).then((user) => {
+            const id = req.body.id;
+            const product_id = req.body.product_id;
+            const list_user_id = user.id;
+            const is_finished = req.body.is_finished;
+            const is_discounted = req.body.is_discounted;
+            const description = req.body.description;
+            const base_price = req.body.base_price;
+            const sql = "INSERT INTO fixed_price (id, product_id, list_user_id, is_finished, is_discounted, description, base_price) VALUES(?, ?, ?, ?, ?, ?, ?)";
+
+            connection.query(sql, [id, product_id, list_user_id, is_finished, is_discounted, description, base_price], (err, results) => {
+              connection.release();
+              if(err) {
+                logger.error("Error adding fixed price auction: \n", err);
+              res
+                .status(400)
+                .send({ success: false, msg: "Error adding fixed price auction" });
+              } else {
+                res
+                .status(200)
+                .send({ success: true, msg: "Fixed price auction successfully created" });
+              }
+            })
+          }).catch(() => {
+            res.status(400);
+          })
+        }
+      })
+  })
+
+  // GET /fixed (get all current fixed options)
+  app.get('/fixed', (req, res) => {
+    pool.getConnection(function (err, connection) {
+      if(err) {
+        // if there is an issue obtaining a connection, release the connection instance and log the error
+        logger.error("Problem obtaining MySQL connection", err);
+        res.status(400).send("Problem obtaining MySQL connection");
+      } else {
+        // if there is no issue obtaining a connection, execute query and release connection
+        const sql = "SELECT * FROM fixed_price";
+        connection.query(sql, (err, rows) => {
+          connection.release();
+          if(err) {
+            logger.error("Error getting fixed price listings: \n", err);
+            res
+              .status(400)
+              .send({ success: false, msg: "Error getting fixed price listings" });
+          } else {
+            res
+              .status(200)
+              .send({ succes: true, data: rows })
```

Comment:
should we calculate the price based on the discount and time here or should we do that on the frontend? maybe we'll need it for the transcation we generate when the listing is done?